hip-config-clang.cmake.in

```diff
@@ -1,4 +1,16 @@
 @PACKAGE_INIT@
+if(NOT DEFINED HSA_PATH)
+    if(NOT DEFINED ENV{HSA_PATH})
+        set(HSA_PATH "/opt/rocm/hsa")
```

Comment:
I'd prefer to add something like `CACHE PATH "Path to which HSA runtime has been installed"` to set command.

Reply:
Added as suggested.